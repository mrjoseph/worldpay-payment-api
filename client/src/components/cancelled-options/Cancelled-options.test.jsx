import React from 'react';
import renderer from 'react-test-renderer';
import CancelledOptions from './Cancelled-options';

describe('CancelledOptions', () => {
  it('Should render the breadcrumb with props', () => {
    const rendered = renderer.create(
      <CancelledOptions />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
