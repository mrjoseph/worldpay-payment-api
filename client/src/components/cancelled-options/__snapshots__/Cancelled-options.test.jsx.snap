// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelledOptions Should render the breadcrumb with props 1`] = `
<div>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <h5
        className="card-title"
      >
        Cancel payment
      </h5>
      <h6
        className="card-subtitle mb-2 text-muted"
      >
        Card subtitle
      </h6>
      <p
        className="card-text"
      >
        Some quick example text to build on the card title and make up the bulk of the card's content.
      </p>
      <a
        className="card-link"
        href="#"
      >
        Cancel payment
      </a>
    </div>
  </div>
</div>
`;
